jax/interpreters/xla.py

```diff
@@ -680,7 +679,7 @@ def _xla_callable(fun: lu.WrappedFun, device, backend, name, donated_invars, *ar
 
   # Computations that only produce constants and/or only rearrange their inputs,
   # which are often produced from partial evaluation, don't need compilation,
-  # and don't need to force their (potentially lazy) arguments.
+  # and don't need to evaluate their arguments.
```

Comment:
Should we just delete this line?

Reply:
Maybe. It's not doing any harm, so I left it in for now. I'm still in two minds about readding C++ level laziness, and if we were to do that you'd still want this "trivial" path most likely.